specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -340,12 +340,26 @@ def verify_kzg_proof_multi_impl(commitment: KZGCommitment,
                                 ys: Sequence[BLSFieldElement],
                                 proof: KZGProof) -> bool:
     """
-    Helper function that verifies a KZG multiproof
+    Verify a KZG multi-evaluation proof for a set of `k` points.
+
+    This is done by checking if the following equation holds:
+        Q(x) Z(x) = f(X) - r(X)
+    Where:
+        f(X) is the polynomial that we want to show opens at `k` points to `k` values
```

Comment:
"we want to show" seems like the wrong PoV here since we are documenting a verification function

Reply:
Changed it to "we want to verify" let me know if that also sounds off